src/js/models/AppModel.js

```diff
@@ -1644,7 +1653,14 @@ define(['jquery', 'underscore', 'backbone'],
               placeholder: "Geographic region",
               icon: "globe",
               description: "The geographic region or study site, as described by the submitter"
-            }
+            },
+            {
```

Comment:
The inline documentation for this config needs to be updated. This is no longer used by the Edit Collection View ever since we updated it to use the new Query Builder. This config will be used in the future when we replace the old DataCatalogView with the DataCatalogWithFilters view. I don't think we want an AnnotationFilter to be a default filter for that view since not all repos will ahve a bioportal API key enabled. So let's remove this here

Reply:
Sounds good. Done in 1987564b73681bc2ca45743e4193aa07a54683d0. I tested that this doesn't break semantic search in the DataCatalogView and it looks fine.